src/components/alert/alert.tsx

```diff
@@ -118,7 +132,13 @@ export class Alert implements OpenCloseComponent {
   //--------------------------------------------------------------------------
 
   connectedCallback(): void {
-    if (this.active && !this.queued) {
+    if ((this.open || this.active) && !this.queued) {
+      if (this.active) {
+        this.activeHandler(this.active);
```

Comment:
I think it'd be clearer and simpler if you use a 'source of truth' var and use it for this whole statement:

```ts
const open = this.open || this.active;
if (open && !this.queued) {
  this.activeHandler(open);
  this.openHandler(open);
}
```

Reply:
neat! 🧴🧽✨